pandas/_libs/lib.pyx

```diff
@@ -942,10 +941,10 @@ _TYPE_MAP = {
     'complex64': 'complex',
     'complex128': 'complex',
     'c': 'complex',
-    'string': 'string' if PY2 else 'bytes',
-    'S': 'string' if PY2 else 'bytes',
-    'unicode': 'unicode' if PY2 else 'string',
-    'U': 'unicode' if PY2 else 'string',
+    'string': 'bytes',
+    'S': 'bytes',
+    'unicode': 'string',
```

Comment:
Perhaps orthogonal but maybe could remove this altogether?

Reply:
Makes sense. There are a couple of `isinstance(..., unicode)` checks but those can be removed in another PR.